app/upgrades/v0.0.10.go

```diff
@@ -0,0 +1,127 @@
+package upgrades
+
+import (
+	"context"
+
+	"cosmossdk.io/math"
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	prooftypes "github.com/pokt-network/poktroll/x/proof/types"
+)
+
+// Upgrade_0_0_10 is the upgrade handler for v0.0.10 Alpha TestNet upgrade
+// Before/after validations should be done using the correct version (e.g. before - v0.0.9, after - v0.0.10)
+var Upgrade_0_0_10 = Upgrade{
+	PlanName: "v0.0.10",
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+		return func(ctx context.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+			//
+			// Add missing parameters and changes from `config.yml`
```

Comment:
I feel like there's a lot more happening in this diff.

- Is the comment just outdated?
- Am I missing something?

Reply:
After reviewing the code further, my understanding is that:
- We use the same diff everywhere
- You grouped things based on comments

In that case: 
- we should put the diff in the function header
- Call out "section migrating new params"...

Reply:
Correct. Changed comments so this is more evident.